### PR TITLE
[NoTicket] Update readme

### PR DESCRIPTION
**Pull request by dharness**
Finally removing that ol' patreon button and a bit of meta-file cleanup

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "react": "15.6.1",
     "react-dom": "15.6.1"
   },
-  "author": "",
+  "author": "kingofthestack",
   "homepage": "",
```
Comment (heatherbooker) [inline package.json]: While we're at it, why not add a homepage? https://kingofthestack.github.io/react-chat-window/

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-chat-window",
-  "version": "1.1.0",
+  "version": "1.2.1",
```
Comment (heatherbooker) [inline package.json]: Which changes are we counting for this release?

File: .gitignore [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-/es
-/lib
-/umd
+es/
+lib/
+umd/
 
 # Node
-/node_modules
+node_modules/
```
Comment (heatherbooker) [inline .gitignore]: Huh, out of curiousity, why the change of slash location?

**Comment by heatherbooker** [comment]
@dharness ^

File: .gitignore (hunk 3 above)
Comment (dharness) [reply .gitignore]: hmm I was having some issues and it was picking up `node_modules`

File: package.json (hunk 2 above)
Comment (dharness) [reply package.json]: Hmm I don't know why this would change, readme changes shouldn't count... I'll undo this

**Comment by dharness** [comment]
@heatherbooker done.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-chat-window",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (dharness) [inline package.json]: whoops....

Comment (heatherbooker) [reply package.json]: But https://www.npmjs.com/package/react-chat-window says 1.2.1 as of one month ago...I don't understand...
